cms/envs/common.py

```diff
@@ -168,7 +168,7 @@
     },
 
     # Teams feature
-    'ENABLE_TEAMS': False,
+    'ENABLE_TEAMS': True,
```

Comment:
I recommend enabling teams through your `lms.env.json` and `cms.env.json` files. That way, the feature flag stays on locally and doesn't show up in version control.